services/ui-src/src/additional-scss/_buttons.scss

```diff
@@ -5,7 +5,8 @@
 .usa-button {
   background: $lochmara;
   width: 140px;
-  box-shadow: 5px 10px 8px #888888;
+  border: none;
+  margin: 0;
 
   &.hollow {
     background-color: $white;
```

Comment:
Honestly we can probably delete all of our `usa-button` styles. We only have one .usa-button on the whole site, and it's the weird non-button in the Quarterly Forms table. If you don't do that in this PR, I'll do it soon in one of mine.